src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -783,6 +783,20 @@ public function prepare_item_for_response( $item, $request ) {
 					}
 
 					$size_data['source_url'] = $image_src[0];
+
+					if ( empty( $data['media_details']['sizes'][ $size ]['sources'] ) || ! is_array( $data['media_details']['sizes'][ $size ]['sources'] ) ) {
```

Comment:
No need to reference the full array address again as this is already accessible via `&$size_data` above.
```suggestion
					if ( empty( $size_data['sources'] ) || ! is_array( $size_data['sources'] ) ) {
```

Reply:
Thanks. This feedback is addressed.